src/Kdyby/Validator/DI/ValidatorExtension.php

```diff
@@ -133,6 +130,7 @@ public function beforeCompile()
 			foreach ((array) $attributes as $name) {
 				$validators[$name] = (string) $service;
 			}
+			$validators[$builder->getDefinition($service)->getClass()] = (string) $service;
```

Comment:
I would pass it through `ReflectionClass` to make sure the name has correct casing.